localstack/cli/main.py

```diff
@@ -1,3 +1,9 @@
+import os
+
+# indicate to the environment we are starting from the CLI
+os.environ["LOCALSTACK_CLI"] = "1"
```

Comment:
This module should only be loaded if the CLI is started, but maybe this should still be moved to the main function body to ensure that this cannot be set accidentally by importing this module?